src/app/components/console/console.cpp

```diff
@@ -83,15 +82,15 @@ void Console::keyPressEvent(QKeyEvent *e)
     }
 }
 
-//void Console::mousePressEvent(QMouseEvent *e)
-//{
-//    Q_UNUSED(e)
-//    setFocus();
-//}
 
+/*
```

Comment:
doxygen style comment
/*! and enter u will see
you code read about it also.
that is for method class enums.